test/parallel/test-util-format.js

```diff
@@ -55,13 +55,16 @@ assert.equal(util.format('%%%s%%%%', 'hi'), '%hi%%');
 })();
 
 // Errors
-assert.equal(util.format(new Error('foo')), '[Error: foo]');
+const err = new Error('foo');
```

Comment:
While you're working in this file, can you change the `require()` statements at the top to use `const` as well?